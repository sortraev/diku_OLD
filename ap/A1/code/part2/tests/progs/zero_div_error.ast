[SExp (Call "print" [Const (StringVal "attempting to divide by zero ...")]),
 SDef "result" (Oper Div (Const (IntVal 42)) (Const (IntVal 0))),
 SExp (Call "print" [Const (StringVal "Successfully divided by zero :)")])]
