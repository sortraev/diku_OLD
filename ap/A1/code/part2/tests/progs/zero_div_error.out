attempting to divide by zero ...
